src/coalesce-vue-vuetify3/src/components/admin/c-admin-editor.vue

```diff
@@ -54,6 +54,11 @@
       </v-btn>
     </v-toolbar>
 
+    <div v-if="metadata.description" class="c-admin-table--description">
```

Comment:
Class name doesn't match this component. Could use a slightly more generic class name if we want to reuse the class on both components.

Reply:
Hm, not really sure what to name it. Looking through the rest of the names it doesn't look like you ever really use the same CSS class across components like this. Do you have a preference? I could also move it to the c-admin-table-page and name it c-admin-description? Also seems a bit weird though.

Reply:
c-admin-page--description